CorsixTH/Lua/rooms/operating_theatre.lua

```diff
@@ -283,24 +281,23 @@ function OperatingTheatreRoom:commandEnteringPatient(patient)
   end
   ----- END Save game compatibility -----
 
-  local room = self
```

Comment:
Normally I agree with removing pointless aliasing, but in this case it may save a sneaky gotcha.

If operation_standby gets called with a : instead of a . then ```self``` would be shadowed by whatever table the callback happened to be a part of. This would be a programming error, but a subtle one that could be hard to catch - whereas room is safe.